<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateSchedule</class>
 <widget class="QMainWindow" name="CreateSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Количество
учебных дней</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="quantityDay">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>7</number>
                </property>
                <property name="value">
                 <number>6</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Размер
популяции</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="sizePopulation">
                <property name="minimum">
                 <number>50</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Кол-во
итераций</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox">
                <property name="minimum">
                 <number>100</number>
                </property>
                <property name="maximum">
                 <number>1000</number>
                </property>
                <property name="value">
                 <number>200</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Факультет
</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="selectFaculty">
              <property name="currentIndex">
               <number>6</number>
              </property>
              <item>
               <property name="text">
                <string comment="101">Биологический</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="102">Географии и геоэкологии</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="103">Исторический</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="104">Математический</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="105">Педагогический</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="106">ИЯиМК</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="107">ПМиК</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="108">Филологический</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="109">Психологии и социальной работы</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="110">Физической культуры</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="111">Физико-технический</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="112">Химико-технологический</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="113">Юридический</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="114">Экономический</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string comment="115">Управления и социологии</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Ход выполнения построения расписания</string>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="format">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="createSchedule">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Начать создание расписания&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/icon/glyphicons-424-git-create.png</normaloff>:/icon/glyphicons-424-git-create.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/icon/glyphicons-517-menu-hamburger.png</normaloff>:/icon/glyphicons-517-menu-hamburger.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
